lib/components/LinearProgressBar/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import clsx from 'clsx';
+import { Text } from '../Text';
+import { TGenericSizes } from "../../types";
+import "./LinearProgressBar.scss"
+
+type TLinearProgressProps = {
```

Comment:
```suggestion
type TLinearProgressProps = ComponentsProp<'div'> & {
```
it would be better to let the consumer pass all applicable props to `div`